extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -75,6 +75,7 @@ function SidebarWithRef(
     side = 'left',
     onBeforeOpen,
     onAfterClose,
+    maskStyle,
```

Comment:
Is the `mask` a good name? It's usually called backdrop... Also, if we supply `backdrop/maskStyle`, then we should also supply `backdrop/maskClassName`.

Reply:
Thanks, updated from `mask` -> `backdrop` throughout and added `backdropClassName` in latest revision.